### PR TITLE
BZ2022076: removed unsupported resources

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.7 and above versions
- [Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=2022076)
- [Preview](https://deploy-preview-43166--osdocs.netlify.app/openshift-enterprise/latest/applications/quotas/quotas-setting-per-project.html#quotas-sample-resource-quota-definitions_quotas-setting-per-project)
@duanwei33 ptal

### CODE REVIEW

**Comment by duanwei33** [comment]
Hi @Amrita42 I think we just need to remove several lines in each yaml:
In compute-resources-long-running.yaml:
    limits.ephemeral-storage: "4Gi" <4>
In compute-resources-time-bound.yaml:
    limits.ephemeral-storage: "1Gi" <4>

**Comment by Amrita42** [comment]
done @duanwei33

File: modules/quotas-sample-resource-quotas-def.adoc [hunk 1]
```diff
@@ -95,15 +95,13 @@ spec:
...
   scopes:
-  - NotTerminating <5>
+  - NotTerminating <4>
 ----
 <1> The total number of pods in a non-terminal state.
 <2> Across all pods in a non-terminal state, the sum of CPU limits cannot exceed this value.
 <3> Across all pods in a non-terminal state, the sum of memory limits cannot exceed this value.
-<4> Across all pods in a non-terminal state, the sum of ephemeral storage limits cannot exceed this value.
-<5> Restricts the quota to only matching pods where `spec.activeDeadlineSeconds` is set to `nil`. Build pods will fall under `NotTerminating` unless the `RestartNever` policy is applied.
+<4> Restricts the quota to only matching pods where `spec.activeDeadlineSeconds` is set to `nil`. Build pods will fall under `NotTerminating` unless the `RestartNever` policy is applied.
```
Comment (nalhadef) [inline modules/quotas-sample-resource-quotas-def.adoc:104]: In the second sentence, delete "will."

File: modules/quotas-sample-resource-quotas-def.adoc [hunk 2]
```diff
@@ -117,15 +115,13 @@ spec:
...
   scopes:
-  - Terminating <5>
+  - Terminating <4>
 ----
 <1> The total number of pods in a terminating state.
 <2> Across all pods in a terminating state, the sum of CPU limits cannot exceed this value.
 <3> Across all pods in a terminating state, the sum of memory limits cannot exceed this value.
-<4> Across all pods in a terminating state, the sum of ephemeral storage limits cannot exceed this value.
-<5> Restricts the quota to only matching pods where `spec.activeDeadlineSeconds >=0`. For example, this quota would charge for build or deployer pods, but not long running pods like a web server or database.
+<4> Restricts the quota to only matching pods where `spec.activeDeadlineSeconds >=0`. For example, this quota would charge for build or deployer pods, but not long running pods like a web server or database.
```
Comment (nalhadef) [inline modules/quotas-sample-resource-quotas-def.adoc:124]: - First sentence: Should there be a space before 0?
- Second sentence: Replace "would charge" with "charges."

**Comment by nalhadef** [comment]
Hi, Amrita. A couple of small comments. I'll mark this LGTM.

File: modules/quotas-sample-resource-quotas-def.adoc (hunk 1 above)
Comment (Amrita42) [reply modules/quotas-sample-resource-quotas-def.adoc:104]: Good point

File: modules/quotas-sample-resource-quotas-def.adoc (hunk 2 above)
Comment (Amrita42) [reply modules/quotas-sample-resource-quotas-def.adoc:124]: The first bullet point is left as is, it already exists and I am not sure If i should change that.
Second one >> implemented

**Comment by mburke5678** [comment]
/enterprise-4.7

**Pull request merged**